### PR TITLE
updated docs around dependencies and virtualenv

### PR DESCRIPTION
**Pull request by indirectlylit**
please review - there may be typos or mistakes :)

re: https://github.com/learningequality/kolibri/pull/837

### CODE REVIEW

File: docs/dev/getting_started.rst [hunk 1]
```diff
@@ -62,46 +63,68 @@ Checking out the code
...
+
+To activate the virtualenv in a standard Bash shell:
+
+.. code-block:: bash
+
+  $ source ~/.venvs/kolibri/bin/activate  # activate the venv
+
+Now, any commands run with ``pip`` will target your virtualenv rather than the global Python installation.
+
+To deactivate the virualenv, run the command below. Note, you'll want to leave it activated for the remainder of project setup!
```
Comment (radinamatic) [inline docs/dev/getting_started.rst]: Change `virualenv` into `virtualenv`.

File: docs/dev/getting_started.rst [hunk 2]
```diff
@@ -62,46 +63,68 @@ Checking out the code
...
-    # Node.js dependencies
-    (kolibri)$ yarn install
-
     # Kolibri Python package in 'editable' mode, so your installation points to your git checkout:
     (kolibri)$ pip install -e .
 
+    # Javascript dependencies
+    (kolibri)$ yarn install
+
+
```
Comment (radinamatic) [inline docs/dev/getting_started.rst:127]: So `pip install -e .` should come before `yarn install`?

**Review by radinamatic — COMMENTED** [review]
Just a small typo!

File: docs/dev/getting_started.rst (hunk 2 above)
Comment (indirectlylit) [reply docs/dev/getting_started.rst:127]: order doesn't matter, but I thought it was cleaner to group all the `pip` actions into one block

**Comment by indirectlylit** [comment]
typo fixed - good catch

**Review by radinamatic — APPROVED** [review]

**Pull request merged**